Formula/llvm.rb

```diff
@@ -432,6 +432,24 @@ def install
 
     # Install Emacs modes
     elisp.install llvmpath.glob("utils/emacs/*.el") + share.glob("clang/*.el")
+
+    return if OS.linux? || !pgo_build
```

Comment:
Probably nice to have a `lto_enabled` variable that could be used throughout in case this ever changes. Can be a separate style PR however.